File: read_xlsx.py
import openpyxl
wb = openpyxl.load_workbook("/Users/levi/Downloads/TMUA_题库_导入格式.xlsx")
ws = wb.active
rows = list(ws.iter_rows(values_only=True))
header = rows[0]
print("总行数(含表头):", len(rows))
print("表头:", header)
for excel_row in [333, 334, 336, 480]:  # 数据行+1(表头占第1行)
    print(f"\n=== Excel 第 {excel_row} 行 (数据行 {excel_row-1}) ===")
    r = rows[excel_row-1]
    for ci, h in enumerate(header):
        print(f"  [{ci}] {h} = {repr(r[ci])}")

File: read_xlsx.mjs
import * as XLSX from 'xlsx';
const wb = XLSX.readFile('/Users/levi/Downloads/TMUA_题库_导入格式.xlsx');
const ws = wb.Sheets[wb.SheetNames[0]];
const rows = XLSX.utils.sheet_to_json(ws, { header: 1, defval: '' });
console.log('总行数(含表头):', rows.length);
console.log('表头:', JSON.stringify(rows[0]));
for (const excelRow of [333, 334, 336, 480]) { // 数据行号+1(表头)
  const idx = excelRow - 1;
  console.log(`\n=== Excel 第 ${excelRow} 行 (数据行 ${excelRow - 1}) ===`);
  console.log(JSON.stringify(rows[idx], null, 0));
  // 逐列打印便于阅读
  rows[0].forEach((h, ci) => {
    console.log(`  [${ci}] ${h} = ${JSON.stringify(rows[idx][ci])}`);
  });
}
